### ISSUE TITLE
[IDEA] Settings - define Default Channel Settings

### ISSUE THREAD

**Issue by Gimzou** [nextgenhealthcare/connect#4864]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to manually set my default parameters for Message Storage and Message Pruning when creating a new Channel.

**Describe your use case**
I need this feature so I can be more productive and make less errors regarding my database storage requirements.

**Describe the solution you'd like**
A way to configure the default values for a New Channel. Maybe in the 'Settings' window under a 'Channel Parameters' tab.
Ex: Set MessageStoreMode to Production instead of Development when I create a New Channel.

**Comment by thorst** [comment]
I agree with having a template. The way we handle this internally is to have a template channel, or example channel if you will, that has all the settings the way we want them for that type of interface. So we have an hl7/tcp/ip template, and a sftp/xml template, etc. Then we clone that channel and rename it and start customizing from there. 

Perhaps what I would enjoy more is a default or global setting. So, we can define the settings for message pruning globally, and then those are used, unless there is an override at the channel level.
